homework-submission/react-hyf/react-hyf-client/src/React_HW1/todoApp.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
```

Comment:
I have 11 warning with the same title `validateDOMNesting(...)` in this route:
Please keep an eye on your console.


<img width="1233" alt="Screen Shot 2019-07-20 at 21 19 57" src="https://user-images.githubusercontent.com/25272822/61583148-5d17ca00-ab34-11e9-939d-5f2c66825ddb.png">

Reply:
I will go through on it.